cmd/apichecker/apicheck.go

```diff
@@ -0,0 +1,1269 @@
+						for {
+							inner := tokenizer.Next()
+							switch inner {
+							case html.EndTagToken:
+								t := tokenizer.Token()
+								if t.Data == htmlData.TokenDataEnd {
+									break loop2
+								}
+							case html.StartTagToken:
+								t := tokenizer.Token()
+								if t.Data == htmlData.TextTokenData {
+									for _, y := range t.Attr {
+										if y.Key == htmlData.Key {
+											r, err := regexp.Compile(htmlData.RegExp)
+											if err != nil {
+												return resp, err
+											}
+											result := r.MatchString(y.Val)
+											if result {
+												resp = append(resp, y.Val)
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// HTMLScrapeYobit gets the check string for Yobit Exchange
+func HTMLScrapeYobit(htmlData *HTMLScrapingData) ([]string, error) {
+	var resp []string
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return resp, err
+	}
+	tokenizer := html.NewTokenizer(temp.Body)
+	var case1, case2, case3 bool
+loop:
+	for {
+		next := tokenizer.Next()
+		switch next {
+		case html.ErrorToken:
+			break loop
+		case html.StartTagToken:
+			token := tokenizer.Token()
+			if token.Data == htmlData.TokenData {
+				for _, x := range token.Attr {
+					if x.Key == htmlData.Key {
+						switch x.Val {
+						case "n4":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								if string(tokenizer.Text()) == "v2" {
+									case1 = true
+								}
+							}
+						case "n5":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr == "v3" {
+									case2 = true
+									resp = append(resp, tempStr)
+								}
+							}
+						case "n6":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr != "v4" {
+									case3 = true
+								}
+								if case1 && case2 && case3 {
+									return resp, nil
+								}
+								resp = append(resp, tempStr)
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// HTMLScrapeLocalBitcoins gets the check string for Yobit Exchange
+func HTMLScrapeLocalBitcoins(htmlData *HTMLScrapingData) ([]string, error) {
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return nil, err
+	}
+	a, err := ioutil.ReadAll(temp.Body)
+	if err != nil {
+		return nil, err
+	}
+	r, err := regexp.Compile(htmlData.RegExp)
+	if err != nil {
+		return nil, err
+	}
+	str := r.FindString(string(a))
+	sha := crypto.GetSHA256([]byte(str))
+	var resp []string
+	resp = append(resp, crypto.HexEncodeToString(sha))
+	return resp, nil
+}
+
+// TrelloGetListsData gets required data for all the lists on the given trello board
+func TrelloGetListsData(idBoard string) ([]ListData, error) {
+	var resp []ListData
+	err := common.SendHTTPGetRequest(pathGetAllLists+idBoard+apiKey+apiToken, true, verbose, &resp)
```

Comment:
This func isn't being used currently and it only supports cmdline vars and not the config ones